wordpress-mobile/WordPress-Android
# Update lint

Updates `line-baseline.xml`: We've fixed bunch of issues and having them in the baseline.xml can hide them if they are re-introduced. We've unfortunately also introduced some new issues, but most of them are just recommendations for better performance. The only "important" issue will be fixed with this PR #8657

I've also increased severity of following issues so we have to explicitly suppress them from now on -  UnusedAttribute, IconXmlAndPng, RtlSymmetry, VectorPath.

To test:
- make sure you can build and run the project
- run ./gradlew lintWasabiDebug -> there should be just 5 warnings and 0 errors 
                 - 3 of those warnings should be fixed in #8657 
                 -  I've left the `GoogleAppIndexingWarning` there so we don't forget about it
                 - the fifth warning just warn us that we are using lint-baseline.xml
